cmd/skopeo/copy.go

```diff
@@ -17,6 +17,9 @@ func copyHandler(context *cli.Context) error {
 
 	policyContext, err := getPolicyContext(context)
 	if err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Errorf("No system policy found. Install the default-policy.json from the source repo: %v", err)
```

Comment:
We're in the process of having a generic skopeo-policy RPM which skopeo will require on installation.

I would drop the "installa the default-policy....."  and point instead to this new rpm. 

@mtrmac wdyt?